src/Constants.h

```diff
@@ -63,6 +63,9 @@ constexpr double EARTH_DIAMETER = EARTH_CIRCUMFERENCE / PI;
  */
 constexpr double EARTHRADIUS = EARTH_DIAMETER / 2;
 
+#define MAX_NUM_THREADS 96
+#define CACHE_LINE_SIZE 64
+
```

Comment:
As you're moving these - make them into `const int` please.

Reply:
Done

Reply:
ping @matt-gretton-dann